service/catalog/impl.go

```diff
@@ -484,7 +484,7 @@ func (a *systemSchemasImpl) Disable(ctx context.Context, request DisableRequest)
 
 func (a *systemSchemasImpl) Enable(ctx context.Context, request EnableRequest) error {
 	path := fmt.Sprintf("/api/2.1/unity-catalog/metastores/%v/systemschemas/%v", request.MetastoreId, request.SchemaName)
-	err := a.client.Do(ctx, http.MethodPost, path, request, nil)
+	err := a.client.Do(ctx, http.MethodPut, path, request, nil)
```

Comment:
What is the context behind Post -> Put?

Reply:
I checked this, and it seems like the wrong HTTP method was used originally but was recently corrected in the OpenAPI spec.

Reply:
I see